assessment.py

```diff
@@ -129,16 +131,52 @@ def POST(self):
         try:
             am = autils.get_assessment_manager()
             form = am.get_bank_form_for_create([])
+            data = self.data()
 
-            form = utilities.set_form_basics(form, self.data())
+            form = utilities.set_form_basics(form, data)
 
             new_bank = utilities.convert_dl_object(am.create_bank(form))
 
+            if 'aliasId' in data:
```

Comment:
an `aliasId` is an alternative `Id` provided by the client, to help identify the `bank`. For example, if I want to be able to pull the bank with an `id` created by Canvas, can I assign it an `aliasId` of `assessment.Bank%3A123456%40CANVAS.COM` (assuming the Canvas internal `id` is `123456`.